Post about recent work towards reviving sandstorm (#310)
sandstorm-io/sandstorm-website: _posts/2020-02-01-reviving-sandstorm.md

---
layout: post
title: "Reviving Sandstorm"
author: Ian Denhardt
authorUrl: https://github.com/zenhack
---

> It's no secret that development on Sandstorm has been pretty sparse

Reviewer: This looks great!

The only thing I'd say is maybe there should be a sentence or two added to the beginning that does two things:

1. Introduce yourself. All the posts in the last three years were written by me, and people might not notice the new authorship if it isn't pointed out.
2. Give a hint that the post is going to end up positive, otherwise people might only read a paragraph or two and assume it's a eulogy. (I know the title is "Reviving Sandstorm", but I think some people won't get the hint from that alone.)

Author: Done.

of a for-profit setting.

Things didn't go as smoothly as we'd hoped, however. While Sandstorm had
had a healthy community of folks using it and building apps, there had
been fewer contributions to Sandstorm itself from outside the company,
and no one but Sandstorm employees had been doing major core development
on a regular basis. Additionally, Kenton has had less time than he'd
hoped to work on Sandstorm.

The project never quite died. It has continued to receive security and
maintenance updates. [Internationalization support was added][2], and
several folks have stepped up to translate the UI into other languages.
A few other features landed, and a few apps continued to see updates as
well. But it would be more than fair to say that the project had stagnated,
and while I and many others were still using it, it was clear that
development wasn't going anywhere fast.

That looks to be changing.

About two months ago, Lyre Calliope sent [an email][3] to the sandstorm-dev
mailing list, starting a discussion on how to get the project moving again.
Since then, I and others have been contributing code and documentation,

> we've been having weekly "office hours" meetings, and in spite of a full

Reviewer 2: ```suggestion
and have had weekly "office hours" meetings. In spite of a full
```

This sentence feels kind of long and run-on as written. In particular, as you start reading it, it feels like "in spite of a full time job" refers to "I and others" rather than Kenton, since you get to his name quite a bit later.
